tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -2109,14 +2110,14 @@ std::unique_ptr<Thunk> IrEmitterUnnested::BuildWhileThunk(
 
   // Generate thunk sequence for while 'condition'.
   HloComputation* condition = hlo->while_condition();
-  IrEmitterUnnested ir_emitter_condition(hlo_module_config_, condition,
-                                         ir_emitter_context_);
+  IrEmitterUnnested ir_emitter_condition(
+      hlo_module_config_, condition, ir_emitter_context_);
```

Comment:
Looks like an unnecessary whitespace change?  It's good to undo these, as they make the diff harder for reviewers to read, and they pollute `git blame`.

If you're using clang-format, you can use the script git-clang-format (found in the LLVM repo) to reformat only the lines you've touched.  Although you now also don't need to worry about it; when we import the patch we'll auto-clang-format it (again, only reformatting lines changed in the PR).

Reply:
Think I might have missed this one inspite of doing a clang-format

Reply:
Hey, I'm still seeing a whitespace change here even though the rest of the line did not change.